libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -28,7 +28,7 @@ ifndef::serverless[]
 endif::[]
 
 To learn about monitoring in general, see 
-{stack-ov}/xpack-monitoring.html[Monitoring the {stack}]. 
```

Comment:
Did this need to change? Wouldn't it be better to go to the more general topic of monitoring the stack instead of monitoring an elasticsearch cluster?

Reply:
The general topic is light on actual useful content: https://www.elastic.co/guide/en/elastic-stack-overview/current/xpack-monitoring.html. It's really just a wrapper for links into the Elasticsearch docs. All the meaty overview content was moved to the Elasticsearch Reference. As a user, I would find it annoying to follow a link that promises to teach me more about monitoring, but instead tells me what I already know and sends me off to other stuff.

Reply:
Fair enough, thanks.